react/finger-exercises/ecmascript/02 - promises/index.js

```diff
@@ -1,9 +1,25 @@
 // Hint: use setInterval, create a new Promise and measure time with Date.now()
 
-export function delay() {
-
+export function delay(time) {
+  let intervalCont = 0;
+  let dateNow = Date.now();
+  return new Promise((resolve, reject) => {
+    if (time > 5000) {
+      const reason = new Error('This time is too much !');
+      reject(reason);
+    }
+    const timer = setInterval(() => {
```

Comment:
try solving this using `setTimeout` instead of `setInterval`, it saves you from doing all the intervalCount logic